owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -155,6 +166,7 @@ class PassCodeActivity : BaseActivity() {
             if (i > 0) {
                 passCodeEditTexts[i]?.setOnKeyListener { v: View, keyCode: Int, _: KeyEvent? ->
                     if (keyCode == KeyEvent.KEYCODE_DEL && bChange) {  // TODO WIP: event should be used to control what's exactly happening with DEL, not any custom field...
+                        passCodeEditTexts[i - 1]?.isEnabled = true
```

Comment:
To avoid repeating the same thing, you can do it like this:
`passCodeEditTexts[i - 1]?.apply{
.isEnabled = true
.setText("")
.requestFocus()
}`

Reply:
Thanks 😃